waku/v2/node/storage/message/sqlite_store/retention_policy_capacity.nim

```diff
@@ -0,0 +1,70 @@
+{.push raises: [Defect].}
```

Comment:
This retention policy would be applicable to all types of stores that we have in the future, right?
Would it be beneficial to have this one directory up?

Reply:
Check my comment: https://github.com/status-im/nwaku/pull/1141#issuecomment-1243721470

Reply:
Absolutely. resolving.